modules/virt-configuring-mtv-providers.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-about-mtv-providers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-mtv-providers_{context}"]
+= Configuring Migration Toolkit for Virtualization providers
+
+(I need to add intro text)
+
+.Procedure
+
+
+[IMPORTANT]
+====
+Question: Then add these MTV docs links?
+Adding an OpenShift Virtualization source provider
+(https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.9/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#adding-source-provider_cnv)
+
+Adding an OpenShift Virtualization destination provider
+(https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.9/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#adding-source-provider_dest_cnv)
+
+Selecting a migration network for an OpenShift Virtualization provider
+(https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.9/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#selecting-migration-network-for-virt-provider_dest_cnv)
```

Comment:
So the migration network is not selected during the configuration of MTV, it is selected during the configuration of OCP-V. In particular I think we should link to the documentation on how to configure a dedicated migration network in OCP-V. The migration network settings in MTV are not relevant for CCLM